downloads/swan-lake-release-notes/swan-lake-alpha2/RELEASE_NOTE.md

```diff
@@ -2,41 +2,599 @@
+    - [Standard Library](#standard-library)
+        - [HTTP Module Improvements](#http-module-improvements)
+        - [MIME Module Improvements](#mime-module-improvements)
+        - [WebSocket Module Improvements](#websocket-module-improvements)
+        - [GraphQL Module Improvements](#graphql-module-improvements)
+        - [WebSub Module Improvements](#websub-module-improvements)
+        - [WebSubHub Module Improvements](#websubhub-module-improvements)
+        - [IO Module Improvements](#io-module-improvements)
+        - [Email Module Improvements](#email-module-improvements)
+        - [TCP Module Improvements](#tcp-module-improvements)
+        - [UDP Module Improvements](#udp-module-improvements)
+        - [Crypto Module Improvements](#crypto-module-improvements)
+        - [JWT Module Improvements](#jwt-module-improvements)
+    - [Code to Cloud](#code-to-cloud)
+    - [Developer Tools](#developer-tools)
+        - [Language Server](#language-server)
+        - [Debugger](#debugger)
+        - [Ballerina Shell REPL - EXPERIMENTAL](#ballerina-shell-repl-experimental)
+    - [Breaking Changes](#breaking-changes)
 
 ### Updating Ballerina
 
-You can use the [update tool](/learn/keeping-ballerina-up-to-date/) to update to jBallerina {{ site.data.stable-latest.metadata.version }} as follows.
+You can use the [update tool](/learn/keeping-ballerina-up-to-date/) to update to Ballerina Swan Lake Alpha2 as follows.
 
-#### For existing users
+#### For Existing Users
 
-If you are already using jBallerina version 1.2.0, or above, you can directly update your distribution to jBallerina {{ site.data.stable-latest.metadata.version }} by executing the following command:
+If you are already using Ballerina, you can directly update your distribution to the Swan Lake channel using the [Ballerina Update Tool](/learn/keeping-ballerina-up-to-date/). To do this, first, execute the command below to get the update tool updated to its latest version. 
 
-> $ ballerina dist update
+> `bal update`
 
-However, you need to use the following commands instead of the above if you have installed:
+If you are using an **Update Tool version below 0.8.14**, execute the `ballerina update` command to update it. Next, execute the command below to update to Swan Lake Alpha2.
 
-- jBallerina 1.2.0 but switched to a previous version: `$ ballerina dist pull jballerina-{{ site.data.stable-latest.metadata.version }}`
-- a jBallerina version below 1.1.0: install via the [installers](/downloads/)
+> `bal dist pull slalpha2`
 
-#### For new users
+#### For New Users
 
-If you have not installed jBallerina, then download the [installers](/downloads/) to install.
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
 
 ### Highlights
 
-### What is new in Ballerina Swan Lake Preview <VERSION>
+- Support for mapping and error binding patterns in the `match` statement
```

Comment:
Shouldn't this be "mapping and error match patterns"? @hasithaa @MaryamZi

Reply:
Ah nvm. Just saw the relevant section. Misunderstood what this says.

Reply:
@pubudu91 - Can we resolve this if this seems fine?